aikau/src/main/resources/alfresco/services/ContentService.js

```diff
@@ -425,6 +427,62 @@ define(["dojo/_base/declare",
       },
 
       /**
+       * Publishes a request to create a form dialog containing a file picker and a location so that the user
+       * can choose what to upload and where to upload it.
+       * 
+       * @instance
+       * @param  {object} payload The payload from the upload request
+       * @since 1.0.34
+       */
+      showUploadLocationPicker: function alfresco_services_ContentService__showUploadLocationPicker(payload) {
+         this.alfPublish("ALF_CREATE_FORM_DIALOG_REQUEST", {
```

Comment:
Why use the hardcoded string here when elsewhere the strings have been replaced with topics constants?